.github/workflows/bundlesize.yml

```diff
@@ -0,0 +1,39 @@
+# Checks bundlesize, does not run on master
+on:
+  pull_request:
+    branches:
+      - '*'
+
+name: Bundlesize
```

Comment:
May need to update something to get workflows to work? Or just get rid of it if hard to modify?

Reply:
okay, just get rid of it